src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,76 @@
+.footer {
+  &__title {
+    color: #1b2129;
+    font-family: "Manrope", sans-serif;
```

Comment:
You have variable for this

However, consider adding default font-family globally